README.md

```diff
@@ -70,6 +70,47 @@ docker run -v $(pwd)/opslevel.yaml:/opslevel.yaml <docker image you built> -c /o
 Configure the ConfigMap and Deploy it to your kubernetes cluster.
 
 ```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: opslevel-payload-check
+rules:
```

Comment:
can you actually change this section to

```yaml
- apiGroups:
    - ""
    - apps
    - extensions
    - batch
    - networking
  resources: 
    - "*"
  verbs:
    - get
    - list
    - watch
```

Reply:
Sure! I was writing specifically for deployments first, but makes sense we would want to open this up to everything.